src/main/scala/org/broadinstitute/dsde/agora/server/dataaccess/AgoraStatus.scala

```diff
@@ -0,0 +1,22 @@
+package org.broadinstitute.dsde.agora.server.dataaccess
+
+import org.broadinstitute.dsde.agora.server.dataaccess.mongo.AgoraMongoClient._
+import org.broadinstitute.dsde.agora.server.dataaccess.permissions.AgoraEntityPermissionsClient._
+
+class AgoraStatus {
+  def status(): (Boolean, String) = {
+    val (up, message) = getMongoDBStatus
```

Comment:
This looks much better, but if the status method just returns the string, then wouldn't it always be a 200 OK even when status is down?